haiku-data/wallpaper_community/wallpaper_community-0.1.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Wallpapers from the Haiku community"
+DESCRIPTION="Wallpapers created by members of the Haiku community"
+HOMEPAGE="https://github.org/haiku/artwork"
+COPYRIGHT="Various Authors"
+LICENSE="
+	CC-BY 4.0
+	CC-BY-SA 4.0
+	"
+REVISION="1"
+srcGitRev="ff46709ba4f5adcdf87621bd1100079cee6e8fde"
+SOURCE_URI="https://github.com/haiku/artwork/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="7921b71930d11b83570669702a09f964cb31de8bda8e2d6db979e46303faab0c"
+SOURCE_DIR="artwork-$srcGitRev"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	wallpaper_community = $portVersion
+	"
+REQUIRES=""
+
+BUILD_REQUIRES=""
+BUILD_PREREQUIRES="
+	cmd:cp
+	cmd:rm"
```

Comment:
For some reason everywhere else but the DESCRIPTION, multi-line end quotes are put in the next empty line with a tab indention. I'm just remarking how the other recipes do it, I'm a bit embarassed to point out these tiny details...

Reply:
Honestly, formatting can be somewhat inconsistent across recipes.